fe1-web/__tests__/model/objects/WalletCryptographyHandler.test.ts

```diff
@@ -0,0 +1,89 @@
+import 'jest-extended';
+import { WalletCryptographyHandler } from 'model/objects/WalletCryptographyHandler';
+import { sign } from 'tweetnacl';
+import STRINGS from 'res/strings';
+import { get } from 'idb-keyval';
+
+/* used to simulate indexedDB database to test store/retrieve functions */
+require('fake-indexeddb/auto');
+
+/* mock crypto library object */
+const crypto = {};
+
+/**
+ * This test uses a MOCK version of the crypto.subtle API.
+ * The scope is not testing the SubtleCrypto library but testing the function
+ * calls to it. It also tests the 'store' and 'retrieve' functions of indexedDB
+ * with an intermediate simulation of app restart.
+ */
+describe('=== Wallet Cryptography Handler tests ===', () => {
+  describe('encryption - decryption', () => {
+    it('should correctly encrypt and decrypt the generated seed', async () => {
+      /* this is the MOCK crypto manager, it uses the mock SubtleCrypto library */
+      // eslint-disable-next-line max-len
+      let mockCryptoManager : WalletCryptographyHandler | null = new WalletCryptographyHandler(crypto);
```

Comment:
if you apply the change I suggested to `WalletCryptographyHandler` (definining `cryptography` as type `Crypto`) you might need to do the following casting:
```suggestion
      let mockCryptoManager : WalletCryptographyHandler | null = new WalletCryptographyHandler(crypto as Crypto);
```